### ISSUE TITLE
NEO Coolcam NAS-PD01 - homey included as a generic device

### ISSUE THREAD

**Issue by Concrete1976** [mruiter/com.neo#223]
Hello,

I received some PIR sensors (NEO Coolcam NAS-PD01) that cannot be correctly include to homey. 
They can only be included as simple zwave device. (generic zwave device)

Homey app version "neo coolcam" is 2.1.16 experimental.

According to the packaging PIR Sensors are NAS-PD01

Information of the generic device in settings:

Vendor ID: 600
Product Type ID: 16
Product ID: 1818
FW:1
Firmware minor version: 5
Firmware ID: 5914
Hardware version 2

Can you please check what´s the problem.

**Comment by koenc** [comment]
I have the same issue; received this new (?) version of the sensor this week. If you like me to ship the sensor to you to add compatibility, please let me know.

**Comment by mruiter** [comment]
Just published a new test version 2.1.17 Please let me know if this version works for you ?Verstuurd vanaf mijn iPhoneOp 2 dec. 2022 om 07:59 heeft Koen Crijns ***@***.***> het volgende geschreven:﻿
I have the same issue; received this new (?) version of the sensor this week. If you like me to ship the sensor to you to add compatibility, please let me know.

**Comment by Concrete1976** [comment]
> I had the same issue; installing version 2.1.16 experimental fixed it!

Thanks, but as you can read in my start comment I'm already on version 2.1.16 experimental.
_"Homey app version "neo coolcam" is 2.1.16 experimental."_

**Comment by mruiter** [comment]
Experimental should be on 2.1.17 Verstuurd vanaf mijn iPhoneOp 2 dec. 2022 om 12:20 heeft Concrete1976 ***@***.***> het volgende geschreven:﻿

I had the same issue; installing version 2.1.16 experimental fixed it!

Thanks, but as you can read in my start comment I'm already on version 2.1.16 experimental.
"Homey app version "neo coolcam" is 2.1.16 experimental."

**Comment by Concrete1976** [comment]
> Experimental should be on 2.1.17 Verstuurd vanaf mijn iPhoneOp 2 dec. 2022 om 12:20 heeft Concrete1976 ***@***.***> het volgende geschreven:﻿ I had the same issue; installing version 2.1.16 experimental fixed it! Thanks, but as you can read in my start comment I'm already on version 2.1.16 experimental. "Homey app version "neo coolcam" is 2.1.16 experimental." —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

@mruiter:
****Thanks** for the effort and the update.** I'll test it as soon as I'm at home.

One more note: I disassembled one of the sensors and it looks like it's a V3 sensor. It can also only be integrated with the V3 method, i.e. by pressing the button for 5 seconds.
I have therefore added the ID to your code, not in the V1 version, but in the V3 version. I added the ID following your logic to the other ID in your code, namely in app.json and in NAS-PD03ZE driver.compose.json. I then transferred the change to my Homey and the sensors could be integrated an work now since 2 days. 
movement = OK
temp. update = OK
brightness update = NOK (a new value is only sent when the battery is replaced) 
That's the only thing that doesn't work properly

When i get home i can upload the user manual for this sensor. According to the description, it can sense movement, temperature, brightness and humidity.
The designation on the circuit board of this sensor is:
NAS-PD01Z1U V3 20210624

I keep you informed.
